webpack/assets/javascripts/react_app/components/UserJwtToken/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { get } from '../../redux/API';
+import { USER_JWT_TOKEN } from './UserJwtTokenConstants';
+import UserJwtTokenForm from './components/UserJwtTokenForm';
+import TokenField from './components/TokenField';
+import { selectStatus, selectUserJwtToken } from './UserJwtTokenSelectors';
+
+const UserJwtToken = ({ userId }) => {
+  const dispatch = useDispatch();
+  const apiStatus = useSelector(selectStatus);
+  const token = useSelector(selectUserJwtToken);
+
+  const [unit, setUnit] = useState('never');
+  const [expValue, setExpValue] = useState('');
+
+  const changeUnit = u => {
+    if (u === 'never') {
+      setExpValue('');
+    }
+    setUnit(u);
+  };
+
+  const handleSubmit = () => {
+    const params = {
+      exp_value: expValue,
+      exp_unit: unit,
+    };
+
+    dispatch(
+      get({
+        key: USER_JWT_TOKEN,
+        url: `/users/${userId}/jwt_token`,
+        params,
+      })
+    );
+  };
+
+  const copyToken = () => {
+    const tmpElement = document.createElement('textarea');
+
+    tmpElement.appendChild(document.createTextNode(token));
+    document.body.appendChild(tmpElement);
+    tmpElement.select();
+    document.execCommand('copy');
+    document.body.removeChild(tmpElement);
+  };
```

Comment:
it would be better to use React ref while having the element exist in the JSX, something like:
```js
  const textAreaRef = useRef(null);

  const copyToClipboard = (e) => {
    textAreaRef.current.select();
    document.execCommand('copy');
  };
```

Reply:
also I wonder if we can achieve something like this:
![Screenshot from 2020-09-01 15-39-52](https://user-images.githubusercontent.com/26363699/91852378-6e089980-ec69-11ea-9218-0df568bf8168.png)

Having the token on one line (I guess user doesn't care much about the whole token string)
and the copy button to its side.
When clicking on the copy there is a small tooltip that sais it was copied :)

Reply:
Is there a component for this or not yet? I couldn't find anything.

Reply:
I don't think there is, but should be easy to implement I guess
maybe after this PR we could move it to common/ components dir.